### PR TITLE
Add onclick functions to Option buttons

### PR DESCRIPTION
**Pull request by emilyzhaoo**
Add onclick functions to option buttons to allow automatic search when clicked by user.

### CODE REVIEW

File: planet/src/main/webapp/places.js [hunk 1]
```diff
@@ -103,6 +103,32 @@ function updateSearch() {
...
+    }
+    document.getElementById('nature').onclick = function() {
+        document.getElementById('pac-input').value = 'nature';
+        setSearchByButton(); 
+    }
+}
+
+/** Search map when option button is clicked */
+function setSearchByButton() {
+    let input = document.getElementById('pac-input');
```
Comment (nabil-dbz) [inline planet/src/main/webapp/places.js]: nit: this can be a const since its value is not changing

File: planet/src/main/webapp/places.js [hunk 2]
```diff
@@ -190,8 +216,3 @@ function listResults(results) {
     // Add search keyword to header
     document.getElementById('greeting').innerHTML = "Find a place: " + document.getElementById('pac-input').value;
```
Comment (nabil-dbz) [inline planet/src/main/webapp/places.js]: nit: single quotes for "Find a place: "

**Review by nabil-dbz — APPROVED** [review]

File: planet/src/main/webapp/places.js [hunk 3]
```diff
@@ -107,6 +107,32 @@ function updateSearch() {
...
+        document.getElementById('pac-input').value = 'nature';
+        setSearchByButton(); 
+    }
+}
+
+/** Search map when option button is clicked */
+function setSearchByButton() {
+    const input = document.getElementById('pac-input');
+    google.maps.event.trigger(input, 'focus', {});
+    google.maps.event.trigger(input, 'keydown', { keyCode: 13 });
```
Comment (EkkiD) [inline planet/src/main/webapp/places.js:136]: what key is this? I'm guessing Enter?

Add a comment saying what these 3 lines are doing and why they're necessary

**Review by EkkiD — CHANGES_REQUESTED** [review]

**Review by EkkiD — APPROVED** [review]
LGTM

**Pull request merged**